Wikipedia/Code/WMFAppViewController.m

```diff
@@ -1486,6 +1491,10 @@ - (void)showExplore {
 #pragma mark - Last Read Article
 
 - (BOOL)shouldShowLastReadArticleOnLaunch {
+    if (!NSUserDefaults.wmf.shouldShowLastReadArticleOnResume) {
+        return NO;
+    }
+
     NSURL *lastRead = [[NSUserDefaults wmf] wmf_openArticleURL];
```

Comment:
Out of scope for this PR but since this is user data it probably shouldn't be in user defaults. `WMFArticle` could have an `isOpen` or `navigationStackIndex` property. The other added benefit there would be the ability to restore the full navigation stack rather than just the top article.

Reply:
Makes sense, it was there before so I didn't touch it

Reply:
Filed separately https://phabricator.wikimedia.org/T221859